pkg/storage/chunk/client/aws/s3_storage_client.go

```diff
@@ -77,6 +88,7 @@ type S3Config struct {
 	SignatureVersion string              `yaml:"signature_version"`
 	SSEConfig        bucket_s3.SSEConfig `yaml:"sse"`
 	BackoffConfig    backoff.Config      `yaml:"backoff_config"`
+	UseIrsa			 bool				 `yaml:"use_irsa"`
```

Comment:
I would prefer to call it `UseIRSA` (on the struct field) just to be explicit about the acronym. (`use_irsa` is fine for yaml).